packages/manager/apps/cloud/client/app/dbaas/logs/detail/inputs/add/edit/logs-inputs-add-edit.controller.js

```diff
@@ -95,9 +96,14 @@ class LogsInputsAddEditCtrl {
     return this.inputAddEdit
       .load()
       .then((successData) => {
-        this.gotToNextStep(this.inputId || successData[0].item.inputId);
+        if (successData[0].item) {
```

Comment:
What is supposed to be in `successData[0]` and in `successData[0].item` ?

Reply:
If related operation succeed, item is given in the promise successData[0]. 
successData[0].item is the LDP operation.
When API reject the call due to a Validation Error, item is not set and so i let the user on the current page to let him fix the incorrect value.

They are probably a better "Lexi" trick to handle that, but it's beyond my skills